app/main.py

```diff
@@ -1,54 +1,21 @@
-import json
-import xml.etree.ElementTree as ET
+from __future__ import annotations
 
+from app.show import BookShow
+from app.serialization import BookSerialization
 
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
-
-    def display(self, display_type: str) -> None:
```

Comment:
No need to change functionality due to task. display method must be available for book